conf/modules.config

```diff
@@ -299,6 +296,42 @@ if (!params.skip_alignment && params.aligner == 'star_salmon'){
             ]
         }
 
+        withName: 'NFCORE_RNASEQ:RNASEQ:.*:BAM_SORT_SAMTOOLS:BAM_STATS_SAMTOOLS:.*' {
```

Comment:
We have to triplicate this block? 😬

Reply:
Ah, you're right. No we don't. They're all in `!params.skip_alignment` blocks. Can make it a single one and just ignore which aligner is being used.

Reply:
Moved it